src/darwin/Framework/CHIP/MTRCertificates.h

```diff
@@ -20,14 +20,17 @@
 
 #import <Foundation/Foundation.h>
 
+typedef NSData MTRCertificateDERBytes;
```

Comment:
IIRC Swift does not pick up typedefs like this correctly, I think for that it would have to be `typedef NSData * MTRCertificateDERBytes;` (and remove the `*` at the usage sites)

Reply:
I can do that.  Thank you for catching that!